stdlib/runtime/compare.gr

```diff
@@ -128,6 +128,18 @@ let rec heapCompareHelp = (heapTag, xptr, yptr) => {
 
       return 0
     },
+    t when t == Tags._GRAIN_UINT32_HEAP_TAG => {
+      let xval = WasmI32.load(xptr, 4n)
+      let yval = WasmI32.load(yptr, 4n)
+      return if (WasmI32.ltU(xval, yval)) -1 else if (WasmI32.gtU(xval, yval)) 1
+      else 0
+    },
+    t when t == Tags._GRAIN_UINT64_HEAP_TAG => {
+      let xval = WasmI64.load(xptr, 8n)
+      let yval = WasmI64.load(yptr, 8n)
+      return if (WasmI64.ltU(xval, yval)) -1 else if (WasmI64.gtU(xval, yval)) 1
+      else 0
```

Comment:
Should probably fix this formatting

Reply:
Let's make an issue.

cc @marcusroberts

Reply:
Did we ever get an issue for this?

Reply:
Yep I made #1654